extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -260,23 +263,20 @@ export class AmpAutocomplete extends AMP.BaseElement {
   getRemoteData_() {
     const ampdoc = this.getAmpDoc();
     const policy = UrlReplacementPolicy.ALL;
-    return batchFetchJsonFor(
-      ampdoc,
-      this.element,
-      /* opt_expr */ undefined,
-      policy
-    ).then(json => {
-      const items = json['items'];
-      if (!items) {
-        user().warn(
-          TAG,
-          'Expected key "items" in data but found nothing. ' +
-            'Rendering empty results.'
-        );
-        return [];
-      }
-      return items;
-    });
+    return batchFetchJsonFor(ampdoc, this.element, 'items', policy)
+      .catch(e => {
+        if (e.message === 'Response is undefined.') {
```

Comment:
Is this the only error we want the component to render empty results for?

Reply:
Before this manually checked if `items` yielded anything before sending the warning, but I realized that the `batchFetchJsonFor` method can check for this directly with the `opt_expr` param. So this is an error that the method is prepared to throw that I did catch for the same reasons as before the change. I'm not sure this handles other errors that can be caused any more than it did before, or if it is a priority to do that in this PR. What do you think?

Reply:
It looks good now, thanks for explaining : ) I don't think we need to look for any other errors. I asked because if we don't want to swallow errors other than when the 'Response is undefined', then we would `throw e` or `return Promise.reject(e)` for errors with other messages. I think the current state is ok